test/Controller/ArticleControllerTest.php

```diff
@@ -1216,12 +1216,13 @@ public function it_displays_a_poa()
      */
     public function it_displays_rds_info_bar_when_it_has_associated_rds()
     {
+        $_ENV['FEATURE_RDS'] = true;
```

Comment:
`@backupGlobals enabled` is needed on any test that manipulates `$_ENV`, otherwise it leaks into other cases.